example/scene_reconstruction/example_scene_reconstruction.py

```diff
@@ -45,12 +45,16 @@ def example_scene_reconstruction(save_dir: str):
         fig.savefig(join(save_dir, fig.get_label() + '.png'))
 
 
-if __name__ == '__main__':
+def example_driver():
```

Comment:
Oh I see, it's because we're looking for methods that start with "example_". With that in mind, how do you feel about "example_main" instead of "example_driver"?

Reply:
We can refactor to example_main, if you prefer.

Reply:
I have a slight preference. Probably not worth another pr just for that, though. 🤷‍♂️